dali/operators/generic/one_hot.cc

```diff
@@ -57,7 +57,11 @@ This value will be cast to the ``dtype`` type.)code", 1.f)
                   R"code(Value that will be used to fill the output to indicate the lack of given
 class in the corresponding input coordinate.
 
-This value will be cast to the ``dtype`` type.)code", 0.f);
+This value will be cast to the ``dtype`` type.)code", 0.f)
+  .AddOptionalArg("layout_axis_name",
+                  R"code(Single character (an upper case letter or a digit) that newly added
+dimension will be named with in the output layout. Provide empty string for empty output
```

Comment:
If you change the default to nullptr you would probably need to adjust the docs here as well.

Reply:
done